### ISSUE TITLE
Some settings/controls are not accessible in Export dialog 

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#20729]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Steps to reproduce

https://github.com/musescore/MuseScore/assets/90187801/2daf91ca-ca2c-45fb-bf06-93407ff0ddc1

### MuseScore Version

4.2

### Regression

Yes, this used to work in a previous version of **MuseScore 4.0.2**

### Operating system

Windows 10

**Comment by DmitryArefiev** [comment]
This was broken in **4.1.0**

**Comment by DmitryArefiev** [comment]
Fixed in #23840
